dev/src/user.clj

```diff
@@ -56,8 +56,7 @@
 
   Try it out: `clj -M:dev:dev-start:drivers:drivers-dev:ee:ee-dev`"
   [& _args]
-  (future (nrepl.cmdline/-main "-p" "50605"))
-  (dev)
+  (future (nrepl.cmdline/-main "-p" "50605" "-b" "0.0.0.0"))
```

Comment:
Binding on 0.0.0.0 instead of localhost makes the repl available in the local network, which is needed for Docker setups.